examples/ITK_Example23_OMEZarr.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    fixed_image = itk.imread('data/CT_3D_lung_fixed.mha', itk.F)
</pre><p>If we use MetaImage format, what does this example have to do with Zarr?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>